source/refined-github.css

```diff
@@ -156,3 +156,8 @@ pr-branches
 	margin-left: 4px !important;
 	margin-right: 4px !important;
 }
+
+/* removes the underline for whitespace on hover, working for issue #4871*/
+.Link--onHover:hover{
```

Comment:
The selector needs to be made more specific to the item we're targeting. I'm not sure if it's possible to target it exactly. Maybe with sibling selectors.

Reply:
yeah, I thought so too. 
I think it can be achieve by going to exact that className from its parent tag's classes. 
A potential solution is this
```
.file-header .file-info .Link--onHover:hover {
	text-decoration: none;
}
```

Reply:
It's better to target `clipboard-copy` instead of more generic `.Link--onHover`.